packages/venia-concept/src/components/CmsBlock/cmsBlock.css

```diff
@@ -0,0 +1,17 @@
+.root {
+    padding: 1rem;
```

Comment:
With multiple CMSBlocks next to each other, I think this should be a margin value rather than padding.

Reply:
Make sense, there are no any left / right paddings / margins in design.